build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -107,12 +112,49 @@ steps:
     displayName: Run unit tests
     condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
 
+  - script: |
+      set -e
```

Comment:
for complicated steps like this it probably makes more sense to just make it a checked in bash file.

Reply:
Yeah makes sense, i'll look into that.